gradle.properties

```diff
@@ -28,3 +28,6 @@ org.gradle.priority=low
 
 # Timeout the daemon after an hour
 org.gradle.daemon.idletimeout=3600000
+
+# dynamic repo-url
+mavenUrl=https://oss.sonatype.org/content/repositories/releases
```

Comment:
https://repo1.maven.org/maven2/ 
?

Should be equivilent, but this is the address usually quoted. this is a good approach though!

Reply:
It's just that I can change the URL when I run the build here at Atruvia. I took the hard-coded URL as default.